src/main/java/com/iota/iri/Snapshot.java

```diff
@@ -157,7 +149,7 @@ public static boolean isConsistent(Map<Hash, Long> state) {
             if (entry.getValue() <= 0) {
 
                 if (entry.getValue() < 0) {
-                    log.info("Skipping negative value for address: " + entry.getKey() + ": " + entry.getValue());
+                    log.info("Skipping negative value for address: %s: %d", entry.getKey(), entry.getValue());
```

Comment:
use format specifiers

Reply:
This won't work. Do: 
                    `log.info("Skipping negative value for address: {}: {}", entry.getKey(), entry.getValue());`